pkgs/tools/networking/openfortivpn/default.nix

```diff
@@ -29,6 +29,6 @@ in stdenv.mkDerivation {
     homepage = "https://github.com/adrienverge/openfortivpn";
     license = stdenv.lib.licenses.gpl3;
     maintainers = [ stdenv.lib.maintainers.madjar ];
-    platforms = stdenv.lib.platforms.linux;
+    platforms = with stdenv.lib.platforms; linux ++ darwin;
```

Comment:
Is there any reason to believe this will be specific to Linux and Darwin? If not, this should be `platforms.unix`.

Reply:
Upstream explicitly recommends only Linux and Darwin: https://github.com/adrienverge/openfortivpn#installing-existing-packages

Maybe it works in other systems, but I don't think so (it needs some specific tooling in Linux and macOS to set the routes after connection).

Reply:
I think it may work outside Linux and macOS if the user set the route theirselves afterwards, but I don't think this is recommended anywhere in documentation (or there is instructions for it).